### DISCUSSION TITLE
Content Guides: is it worth it to add more layers of info in the notes/warning callouts?

### DISCUSSION THREAD

**Discussion by tmgonzales** [idom-team/idom#1044] [Ideas]
From the perspective of our learners, is it worth it to expand the information of the notes & warning callouts? Adding in examples of incorrect and correct usage - the example below is going through scenarios of adding in a photo component

[Reactpy.dev / Docs / Guides / Creating Interfaces / Your First Components](https://reactpy.dev/docs/guides/creating-interfaces/your-first-components/index.html#defining-a-component)

```python
class ComponentType:
    def __init__(self, name):
        self.name = name

def some_function(component):
    if not isinstance(component, ComponentType):
        raise TypeError('Expected a ComponentType, not {}'.format(type(component).__name__))
    # rest of function...

# Using the function correctly
c = ComponentType('example')
some_function(c)

# Using the function incorrectly
d = {'name': 'example'}
some_function(d)  # This will raise the TypeError

```

**Comment by Archmonger** [comment]
Yes this is intended. See #863
